codesign/eclipse-notarize.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+################################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -eu
+
+PKG="$1"
+PRIMARY_BUNDLE_ID="$2"
+
+echo "Notarizing $1, this can take a while!"
```

Comment:
Might be worth adding that you'll update status every minute

Reply:
done